src/test/java/com/kodilla/ecommercee/domain/GroupEntityTests.java

```diff
@@ -0,0 +1,86 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+
+@SpringBootTest
+@Transactional
+public class GroupEntityTests {
+
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Test
+    public void testSaveGroup() {
+
+        //Given
+        Group group = new Group();
+
+        Product product = new Product();
+        List<Product> productList = new ArrayList<>();
+        productList.add(product);
+        productRepository.save(product);
+        group.setProducts(productList);
+        group.setName("Test Group");
+
+        //When
+        groupRepository.save(group);
+        Long groupId = group.getGroupId();
+
+        //Then
+        Assertions.assertNotEquals(0, groupId);
```

Comment:
To check if an object has been created, I would personally use `assertTrue `and the `existsById `method from the repository. However, this is just a matter of code clarity.

Reply:
I will try. Maybe I will add next line.